src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -488,6 +522,10 @@
           [rn/view
            {:test-ID                       "record-audio"
             :style                         style/button-container
+            :hit-slop                      {:top    (if platform/ios? -70 0)
```

Comment:
We reduce touchable area for the record audio component in iOS so text input and record audio component can live together without overlapping touches on invisible areas where this component overlaps text input and text input should respond to touches unless a recording audio is in progress (that case is handled by onResponderMove while recording).
On Android we don't need this because it seems touches are handled differently and work as expected.